modules/dbnd/src/dbnd/_core/task_ctrl/task_visualiser.py

```diff
@@ -40,7 +41,7 @@ def operators_list_str(operators, separator="\n\t"):
     "task_band",
 ]
 
-_MAX_VALUE_SIZE = 1500
+_MAX_VALUE_SIZE = config.get_config_value("describe", "task_visualiser_max_value_size")
```

Comment:
our config is not "static" like in airflow, 
airflow confgig loaded on import, ours during runtime. 

we better do  from TaskVisualizer :   self.task.settings.describe. task_visualiser_max_value_size. (may be i am wrong with exact names)

Reply:
fixd